.github/workflows/webcomponents.yml

```
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - '*.*.*'
```

```suggestion
      - 'v*.*.*'
```

That might make things a bit safer?